pinot-server/src/test/java/org/apache/pinot/server/integration/realtime/RealtimeTableDataManagerTest.java

```diff
@@ -59,6 +59,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
```

Comment:
This import never gets used though. 
But since you're gonna delete this test, it should be good here.